gsuite/resource_group_members.go

```diff
@@ -0,0 +1,249 @@
+package gsuite
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	directory "google.golang.org/api/admin/directory/v1"
+)
+
+func resourceGroupMembers() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGroupMembersCreate,
+		Read:   resourceGroupMembersRead,
+		Update: resourceGroupMembersUpdate,
+		Delete: resourceGroupMembersDelete,
+
+		Schema: map[string]*schema.Schema{
+			"group": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"member": {
+				Type:     schema.TypeSet,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: schemaMember,
+				},
+			},
+		},
+	}
+}
+
+func resourceGroupMembersRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	members, err := getApiMembers(d.Id(), config)
+
+	if err != nil {
+		return err
+	}
+
+	d.Set("group", d.Id())
+	d.Set("member", membersToCfg(members))
```

Comment:
`gsuite/resource_group_members.go:45:30: cannot use members (type []*admin.Member) as type *admin.Members in argument to membersToCfg`